ligand/models.py

```diff
@@ -311,188 +339,280 @@ def __str__(self):
+    standard_value = models.CharField(max_length=10, null=True)
     standard_relation = models.CharField(max_length=10)
     standard_type = models.CharField(max_length=20)
     standard_units = models.CharField(max_length=20)
 
-    publication = models.ForeignKey(Publication, on_delete = models.CASCADE, null= True)
-    chembl = models.CharField(max_length=100,null = True)
-    smiles = models.TextField(null = True)
-    activity = models.CharField(max_length=50,null = True)
-    document_chembl_id = models.CharField(max_length=50,null = True)
-    cell_line = models.TextField(null = True)
+    publication = models.ForeignKey(
+        Publication, on_delete=models.CASCADE, null=True)
+    chembl = models.CharField(max_length=100, null=True)
+    smiles = models.TextField(null=True)
+    activity = models.CharField(max_length=50, null=True)
+    document_chembl_id = models.CharField(max_length=50, null=True)
+    cell_line = models.TextField(null=True)
+
 
 class LigandVendors(models.Model):
     slug = models.SlugField(max_length=100, unique=True)
     name = models.CharField(max_length=200, default='')
     url = models.TextField(null=True)
 
+
 class LigandVendorLink(models.Model):
     vendor = models.ForeignKey('LigandVendors', on_delete=models.CASCADE)
-    lp = models.ForeignKey('LigandProperities', related_name='vendors', on_delete=models.CASCADE)
-    url = models.CharField(max_length=300)  #SourceRecordURL
-    vendor_external_id = models.CharField(max_length=300) #RegistryID
-    sid = models.CharField(max_length=200, unique=True) #SID
+    lp = models.ForeignKey('LigandProperities',
+                           related_name='vendors', on_delete=models.CASCADE)
+    url = models.CharField(max_length=300)  # SourceRecordURL
+    vendor_external_id = models.CharField(max_length=300)  # RegistryID
+    sid = models.CharField(max_length=200, unique=True)  # SID
+
+# Biased Signalling - start
+
 
-#Biased Signalling - start
 class BiasedExperiment(models.Model):
     submission_author = models.CharField(max_length=50)
     ligand = models.ForeignKey(Ligand, on_delete=models.CASCADE)
     publication = models.ForeignKey(Publication, on_delete=models.CASCADE)
     receptor = models.ForeignKey('protein.Protein', on_delete=models.CASCADE)
-    endogenous_ligand = models.ForeignKey(Ligand, related_name='endogenous_ligand_bias', on_delete = models.CASCADE,  null=True)
-    auxiliary_protein = models.ForeignKey('protein.Protein', on_delete=models.CASCADE, related_name='auxiliary_protein', null=True)
-    ligand_source_id = models.TextField(null = True)
-    ligand_source_type = models.TextField(null = True)
-
+    endogenous_ligand = models.ForeignKey(
+        Ligand, related_name='endogenous_ligand_bias', on_delete=models.CASCADE,  null=True)
+    auxiliary_protein = models.TextField(null=True)
+    ligand_source_id = models.TextField(null=True)
+    ligand_source_type = models.TextField(null=True)
 
 class ExperimentAssay(models.Model):
```

Comment:
Can you possibly rename this to BiasedAssayExperiment to not be confused with the AssayExperiment table